plutarch-test/src/Plutarch/BoolSpec.hs

```diff
@@ -0,0 +1,64 @@
+module Plutarch.BoolSpec (spec) where
```

Comment:
This isn't your fault, but GH doesn't syntax-highlight this file for some reason...

Reply:
🤷🏿 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/3998/154166989-cc34e5d2-b9fe-4d8a-bf02-ead28acb21f3.png">